app/cdap/components/NamespaceAdmin/ServiceAccounts/EditConfirmDialog.tsx

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { useEffect, useState } from 'react';
+import T from 'i18n-react';
+import styled from 'styled-components';
+import TextField from '@material-ui/core/TextField';
+import { ConfirmDialog, SeverityType } from 'components/shared/ConfirmDialog';
+import {
+  validateServiceAccount,
+  addServiceAccount,
+} from 'components/NamespaceAdmin/store/ActionCreator';
+
+const PREFIX = 'features.ServiceAccounts';
+const GCLOUD_COMMAND_TEXT =
+  'gcloud iam service-accounts add-iam-policy-binding \\\n' +
```

Comment:
could you just back tick `` this command instead?

Reply:
@seanfeld @radhikav1 
Here, the GCLOUD_COMMAND_TEXT is the string that the user needs to copy and paste in a terminal. Therefore, we prefer to use the `${ VARIABLE_NAME }` syntax in the string literal itself, so that the user can make use of the environment variables in their terminal session.

we could use template literals and write the same functionality like the following

```js
/**
 * Generates the gcloud cli command to add an IAM policy binding. If any of the
 * parameters for the command is not provided when the command is generated, then
 * the user should be able to provide them as environment variables in their shell.
 *
 * @param  tenantProjectId string, defaults to "${TENANT_PROJECT_ID}" so it can be
 *         provided as the environment variable TENANT_PROJECT_ID when
 *         the command is run
 * @param  identity string, defaults to "${IDENTITY}" so that it can be provided as the
 *         environment variable IDENTITY when the command is run
 * @param  gsaEmail string, defaults to "${GSA_EMAIL}" so that it can be provided as the
 *         environment variable GSA_EMAIL when the command is run
 * @return string, the gcloud cli command to run
 */
const getGcloudCommand = ({
  tenantProjectId?: string = "${TENANT_PROJECT_ID}",
  identity?: string = "${IDENTITY}", 
  gsaEmail?: string = "${GSA_EMAIL}"
}): string =>
  `gcloud iam service-accounts add-iam-policy-binding \
    --role roles/iam.workloadIdentityUser \
    --member "serviceAccount:${tenanatProjectId}.svc.id.goog[default/${identity}]" \
    ${gsaEmail}`
```

and use this function to get the actual command to display (to be copied)  in line 77 of this PR, avoiding the `string.replace` usage there. But the above code snippet does not seem less confusing either.

Reply:
I would still recommend the above snippet, as it more concise and offers better readability over the current approach.